File: app/front/src/components/main/dto/riyousha/getRiyoushaPartnerApiByEntityCapsuleDto.ts
import { FrameworkCapsuleDto, type FrameworkCapsuleDtoInterface } from "seijishikin-jp-normalize_common-tool";
import { RiyoushaPartnerApiMasterEntity, type RiyoushaPartnerApiMasterEntityInterface } from "../../entity/riyoushaPartnerApiMasterEntity";

interface GetRiyoushaPartnerApiByEntityCapsuleDtoInterface extends FrameworkCapsuleDtoInterface {

    /** 利用者APIパートナーマスタEntity */
    masterEntity: RiyoushaPartnerApiMasterEntityInterface;
}

class GetRiyoushaPartnerApiByEntityCapsuleDto extends FrameworkCapsuleDto implements GetRiyoushaPartnerApiByEntityCapsuleDtoInterface {

    /** 利用者APIパートナーマスタEntity */
    masterEntity: RiyoushaPartnerApiMasterEntityInterface;

    constructor() {
        super();
        this.masterEntity = new RiyoushaPartnerApiMasterEntity();
    }
}

export { type GetRiyoushaPartnerApiByEntityCapsuleDtoInterface, GetRiyoushaPartnerApiByEntityCapsuleDto }

File: app/back/src/main/java/net/seijishikin/jp/normalize/manage/kanrensha/dto/riyousha/GetRiyoushaPartnerApiByEntityCapsuleDto.java
package net.seijishikin.jp.normalize.manage.kanrensha.dto.riyousha;

import java.io.Serializable;

import net.seijishikin.jp.normalize.common_tool.dto.FrameworkCapsuleDto;
import net.seijishikin.jp.normalize.manage.kanrensha.entity.RiyoushaPartnerApiMasterEntity;

/**
 * マスタEntityから利用者APIパートナー編集用利用者取得条件Dto
 */
public class GetRiyoushaPartnerApiByEntityCapsuleDto extends FrameworkCapsuleDto implements Serializable {

    /** Serialize id */
    private static final long serialVersionUID = 1L;

    /** 利用者APIパートナーマスタEntity */
    private RiyoushaPartnerApiMasterEntity masterEntity;

    /**
     * 利用者APIパートナーマスタEntityを取得する
     * 
     * @return 利用者APIパートナーマスタEntity
     */
    public RiyoushaPartnerApiMasterEntity getMasterEntity() {
        return masterEntity;
    }

    /**
     * 利用者APIパートナーマスタEntityを設定する
     * 
     * @param masterEntity 利用者APIパートナーマスタEntity
     */
    public void setMasterEntity(final RiyoushaPartnerApiMasterEntity masterEntity) {
        this.masterEntity = masterEntity;
    }

}
